includes/class-wcsatt-core-compatibility.php

```diff
@@ -48,7 +55,7 @@ private static function get_wc_version() {
 	 * @return boolean
 	 */
 	public static function is_wc_version_gte_2_7() {
```

Comment:
> Are we dropping support for WC 2.6, after all?

I'm keen to, but letting you make the call on what you're most comfortable doing. :)